### ISSUE TITLE
Request: add (×) multiplication operator to Prelude.Unicode

### ISSUE THREAD

**Issue by unclechu** [roelvandijk/base-unicode-symbols#20]
```haskell
{-|
(&#x00D7;) = ('*')

U+00D7, MULTIPLICATION SIGN
-}
(×) ∷ Num α ⇒ α → α → α
(×) = (*)
{-# INLINE (×) #-}
```

**Comment by unclechu** [comment]
@roelvandijk I've created a PR for https://github.com/roelvandijk/base-unicode-symbols/pull/21

**Comment by roelvandijk** [comment]
Thank you for the pull request. It has been merged.

**Comment by unclechu** [comment]
@roelvandijk Are you planning to make a release soon?

**Comment by unclechu** [comment]
It's offtopic but you also forgot to add a version tag to git-repo https://github.com/roelvandijk/base-unicode-symbols/releases (in cabal and hackage it's `0.2.3` but latest release is `0.2.2.4`).

**Comment by roelvandijk** [comment]
I just released version [0.2.4](https://hackage.haskell.org/package/base-unicode-symbols-0.2.4).

I also added tags for both [0.2.3](https://github.com/roelvandijk/base-unicode-symbols/releases/tag/0.2.3) and [0.2.4](https://github.com/roelvandijk/base-unicode-symbols/releases/tag/0.2.4). Thanks for the heads up about the version tag.
